haystack/nodes/answer_generator/openai.py

```diff
@@ -112,6 +165,8 @@ def __init__(
         self.examples_context = examples_context
         self.examples = examples
         self.stop_words = stop_words
+        self.instruction_prompt = prompt_template
```

Comment:
would name it here also `self.prompt_template` to avoid confusion if these are two different things